File: ritexe-backend/models/answer.py
from sqlalchemy import Integer, Table,Column,String,DateTime
from config.db import meta,engine

answers = Table('answer',meta,
Column('ans_id',Integer,primary_key=True),
Column('ans',String(255)),
Column('upvote',Integer),
Column('downvote',Integer),)
# Column('date',DateTime)


meta.create_all(engine)

File: ritexe/lib/models/answer.dart
class Answer {
  final int ansId, upVote, downVote, qsId, userId;
  final String ans;
  final bool isUpvoted, isDownvoted;
  Answer(
      {required this.ans,
      this.isUpvoted = false,
      this.isDownvoted = false,
      this.ansId = 0,
      this.downVote = 0,
      required this.qsId,
      this.upVote = 0,
      required this.userId});
}
